doc/using/InvokingGHDL.rst

```diff
@@ -624,64 +627,64 @@ have been split for readability):
       [std_logic_vector, std_logic_vector return boolean]
   ../translate/ghdldrv/ghdl: compilation error
 
-Indeed, the `"="` operator is defined in both packages, and both are visible at the place it is used.  The first declaration is an implicit one, which occurs when the `std_logic_vector` type is declared and is an element to element comparison, the second one is an explicit declared function, with the semantic of an unsigned comparison.
+Indeed, the `"="` operator is defined in both packages, and both are visible at the place it is used.  The first declaration is an implicit one, which occurs when the `std_logic_vector` type is declared and is an element to element comparison.  The second one is an explicit declared function, with the semantics of an unsigned comparison.
 
-With some analyser, the explicit declaration has priority over the implicit declaration, and this design can be analyzed without error.  However, this is not the rule given by the VHDL LRM, and since GHDL follows these rules,
+With some analysers, the explicit declaration has priority over the implicit declaration, and this design can be analyzed without error.  However, this is not the rule given by the VHDL LRM, and since GHDL follows these rules,
 it emits an error.
 
-You can force GHDL to use this rule with the *-fexplicit* option (see :ref:`GHDL_options` for further details). However it is easy to fix this error, by using a selected name:
+You can force GHDL to use this rule with the *-fexplicit* option (see ':ref:`GHDL:options`' for further details). However it is easy to fix this error, by using a selected name:
 
 .. code-block:: VHDL
 
-  library ieee;
-  use ieee.std_logic_unsigned.all;
-
-  architecture fixed_bad of counter
-  is
-     signal v : std_logic_vector (3 downto 0);
-  begin
-     process (ck, rst)
-     begin
-       if rst = '1' then
-          v <= x"0";
-       elsif rising_edge (ck) then
-          if ieee.std_logic_unsigned."=" (v, "1010") then
-             v <= x"0";
-          else
-             v <= v + 1;
-          end if;
-       end if;
-     end process;
-
-     val <= v;
-  end fixed_bad;
-
-It is better to only use the standard packages defined by IEEE, which provides the same functionalities:
+   library ieee;
+   use ieee.std_logic_unsigned.all;
+
+   architecture fixed_bad of counter
+   is
+      signal v : std_logic_vector (3 downto 0);
+   begin
+      process (ck, rst)
+      begin
+         if rst = '1' then
```

Comment:
Why has indentation changed?

Reply:
The first example of a code block I found was in the [Quick Start Guide](https://ghdl.readthedocs.io/en/latest/using/QuickStartGuide.html). It used a 3-space indent. Therefore, I standardized all other documents to use a 3-space indent inside code block examples.

Reply:
stackoverflow.com/questions/48311159/is-3-space-indentation-required-in-rest

Reply:
@1138-4EB This is source code in a code block, not ReST code itself.

The 3-space indentation is going back to Ada and the origins of VHDL. I highly dislike it because it is not a power of 2.

@tgingold thoughts? I think you are using 3-space indentation in your Ada source files, what about the VHDL examples in the documentation?

Reply:
I use two spaces in my own VHDL code, so I'm ok with it.

Just to make it clear, we should keep using three spaces in RST sources unless it is inside a code blocl, shouldn't we?

Reply:
Yes.